kbfsfuse/main.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/keybase/kbfs/libfs"
 	"github.com/keybase/kbfs/libfuse"
 	"github.com/keybase/kbfs/libkbfs"
+
+	_ "github.com/songgao/stacktraces/on/SIGUSR2"
```

Comment:
Intentionally committed?  I'm not against it, we already have a way to get goroutines without killing the process: `/keybase/.kbfs_profiles/goroutine`.